src/wp-includes/load.php

```diff
@@ -730,7 +730,7 @@ function wp_start_object_cache() {
 	}
 
 	if ( function_exists( 'wp_cache_add_global_groups' ) ) {
-		wp_cache_add_global_groups( array( 'users', 'userlogins', 'usermeta', 'user_meta', 'useremail', 'userslugs', 'site-transient', 'site-options', 'blog-lookup', 'blog-details', 'site-details', 'rss', 'global-posts', 'blog-id-cache', 'networks', 'sites', 'blog_meta' ) );
+		wp_cache_add_global_groups( array( 'users', 'userlogins', 'usermeta', 'user_meta', 'useremail', 'userslugs', 'site-transient', 'site-options', 'blog-lookup', 'blog-details', 'site-details', 'rss', 'global-posts', 'blog-id-cache', 'networks', 'sites', 'blog_meta', 'site_meta' ) );
```

Comment:
I thought `blog_meta` was used for site meta for consistency with the database table name. Please correct me if I am confused about names.

Reply:
> Please correct me if I am confused about names.

Everyone is confused about multisite names because multisite names confuse everyone 🤣

In this specific instance, `blog_meta` refers to the `wp_blogmeta` database table via the meta API, hence `site_meta` will similarly refer to the `wp_sitemeta` database table.

meta.php gathers the cache-group name via a simple concatenation:
```php
$meta_type . '_meta'
```

Because this PR is dropping the single "network options" approach for the "per network meta" one, there is no cache group to drop, and 1 to add.

Reply:
Thanks John!

I think I have it: blogs are sites, sites are networks, naming things remains difficult.